lib/player.js

```diff
@@ -1737,6 +1737,12 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
           }, 'initializeMediaSourceEngineInner_');
         }
 
+        if (this.manifest_ && this.manifest_.textStreams.length) {
+          if (this.textDisplayer_.enableTextDisplayer) {
```

Comment:
This is API change, so we should log info we expect this method now.

Reply:
Done!